submission/notifications/src/main/kotlin/ebi/ac/uk/notifications/util/TemplateLoader.kt

```diff
@@ -1,12 +1,30 @@
 package ebi.ac.uk.notifications.util
 
+import ebi.ac.uk.extended.model.ExtSubmission
+import ebi.ac.uk.util.collections.firstOrElse
 import org.apache.commons.io.IOUtils
 import org.springframework.core.io.ResourceLoader
 import java.nio.charset.Charset
 
+internal const val DEFAULT_TEMPLATE = "Default"
+
 class TemplateLoader(private val resourceLoader: ResourceLoader) {
     fun loadTemplate(templateName: String): String {
-        val resource = resourceLoader.getResource("classpath:templates/$templateName")
-        return IOUtils.toString(resource.inputStream, Charset.defaultCharset())
+        val template = resourceLoader.getResource("classpath:templates/$templateName")
+        return IOUtils.toString(template.inputStream, Charset.defaultCharset())
+    }
+
+    fun loadCollectionTemplate(sub: ExtSubmission, templateName: String): String {
```

Comment:
this one gives the idea that every collection has a template, maybe `loadTemplateOrDefault`